natterstefan/nextjs-template
# fix: write version to version.js with node, simplify setup

# TASK

## What I did

- `version.js` is written with node instead of `gradlew` now
- resolves https://github.com/natterstefan/nextjs-template/issues/6

## How to test



- run `yarn` and you should see the setup output
- watch `version.js` for changes

## Screenshots



404
